packages/gatsby/src/recipes/README.md

```diff
@@ -20,9 +20,31 @@ Follow the issue for updates!
 
 https://github.com/gatsbyjs/gatsby/issues/22991
 
-## How to write a recipe
 
-Let's write our first Recipe to setup Emotion on a Gatsby site.
+## Running an example recipe
+First, try running a recipe called `gatsby recipes emotion` by following these steps:
+
+Create a new Hello World Gatsby site:
+
+`gatsby new try-emotion https://github.com/gatsbyjs/gatsby-starter-hello-world`
+
+Meanwhile also upgrade the global gatsby-cli package to use the one with new recipes command.
+
+`npm install -g gatsby-cli@recipes`
+
+Once the global finishes installing, running `gatsby -v` should show this as your version: `Gatsby CLI version: 2.11.4-recipes.53`
+
+Upgrade the version of Gatsby installed to use one with Recipes.
+
+`yarn add gatsby@recipes`
+
+Now run the Emotion recipe and follow the commands.
+
+`gatsby recipes emotion`
+
+## Writing a recipe
```

Comment:
Alternate headings: The ingredients of a recipe, The architecture of a recipe, The contents of a recipe, what is in a recipe?

Reply:
This is also resolved since @laurieontech's commits took care of it 18ce1fb